### PR TITLE
feature/FOUR-15127

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
When the process is collapsed it show all the description of the process 

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-15127

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-208794c5c6.engk8s.processmaker.net

File: resources/js/processes-catalogue/components/ProcessCollapseInfo.vue [hunk 1]
```diff
@@ -185,6 +185,11 @@ export default {
     activateReadMore() {
       this.readActivated = true;
     },
+    getNameEllipsis() {
+      const name = this.process.name;
+      const nameEllipsis = name.slice(0, 50);
+      return nameEllipsis + "..."
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/ProcessCollapseInfo.vue:191]: for the short name the three dots will always be present

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-208794c5c6.engk8s.processmaker.net

File: resources/js/processes-catalogue/components/ProcessScreen.vue [hunk 2]
```diff
@@ -128,6 +128,12 @@ export default {
           this.showScreen = response.data.config !== null;
         });
     },
+    getNameEllipsis() {
+      const name = this.process.name;
+      console.log(name.length);
```
Comment (pmPaulis) [inline resources/js/processes-catalogue/components/ProcessScreen.vue:133]: remove this console.log

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-208794c5c6.engk8s.processmaker.net

**Review by fagubla — APPROVED** [review]

File: resources/js/processes-catalogue/components/ProcessScreen.vue [hunk 3]
```diff
@@ -128,6 +128,11 @@ export default {
           this.showScreen = response.data.config !== null;
         });
     },
+    getNameEllipsis() {
+      const name = this.process.name;
+      const nameEllipsis = name.length <= 70 ? name : name.slice(0, 70) + "...";
+      return nameEllipsis;
+    }
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/ProcessScreen.vue:135]: add this method to the mixin file

**Review by fagubla — CHANGES_REQUESTED** [review]

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-208794c5c6.engk8s.processmaker.net

**Review by CarliPinell — APPROVED** [review]

**Review by pmPaulis — APPROVED** [review]

**Pull request merged**